lib/autoproj/cli/base.rb

```diff
@@ -176,11 +176,13 @@ def self.validate_options(args, options)
                     debug: false,
                     color: TTY::Color.color?,
                     progress: TTY::Color.color?,
-                    parallel: nil
+                    parallel: nil,
+                    interactive: nil
 
                 Autoproj.silent = options[:silent]
                 Autobuild.color = options[:color]
                 Autobuild.progress_display_enabled = options[:progress]
+                Autoproj.interactive = options[:interactive]
```

Comment:
The only purpose of this is to pass the CLI argument to the config object. Just set the argument (if given) to the config object instead, removing the middle-man.

All these global flags have their history ... but they're meant to disappear slowly ...

Reply:
Shifted the flag into the Autoproj::Configuration namespace if that is what you intended. Using an instance variable fails, since the Autoproj.config object seems to be replaced at a later stage.

Reply:
`Autoproj.config` is really only meant for the autobuild files. A workspace config is available through the config object (`ws.config` in CLI classes). If it doesn't work, I can help and debug it. The bottom line being that I'd rather not have yet-another global added.